### ISSUE TITLE
help: cannot reproduce segmentation results

### ISSUE THREAD

**Issue by celeste-1** [valeoai/SLidR#1]
Thank you for releasing this code. The idea is awesome! However, there may be something wrong when I try to reproduce the nuscenes segmentation result. Could you give me some help?  I followed your tutorials from the very beginning and used the pretrain model provided, but got only **0.360** [it is supposed to be **0.383**] mIoU when using 1% of the annotated training scans with 100 epochs. The detailed results are show in fig. The environment is the required version.
![1649685387(1)](https://user-images.githubusercontent.com/43608840/162755367-f6289e14-96dd-40c0-b9f8-1038f3a01949.png)

**Comment by CSautier** [comment]
Hi, thank you for your interest, there is indeed a problem, it seems to be with the learning rate as we mention in the supplementary "We use different learning rates on the classification head and the backbone fθ" but that does not appear in the config I pushed. I am currently re-training a version with the learning rates we used to obtain the 38.3 and if I can reproduce it, I will add in the README a section about all the final learning rates found for every experiment in the paper and how to use them. (The config as they are now reproduce the from scratch results)

**Comment by CSautier** [comment]
I was able to reproduce the results in the paper.
You need to modify the learning rates in the config with
lr : 0.02
lr_head : 2.0
I will modify the code and the README accordingly

**Comment by Stephanie-Shen324** [comment]
@CSautier Hi, would you mind providing the lr for random initialisation? for both kitti & nuscenes dataset. Thank you!
